pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -361,7 +362,26 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Elasticsearch gateway certificate are not available yet, waiting until they become available", nil, reqLogger)
 		return reconcile.Result{}, nil
 	}
-	trustedBundle := certificateManager.CreateTrustedBundle(managerInternalTLSSecret, esgwCertificate)
+	//trustedBundle := certificateManager.CreateTrustedBundle(managerInternalTLSSecret, esgwCertificate)
+
+	linseedCertLocation := render.TigeraLinseedSecret
+	linseedCertificate, err := certificateManager.GetCertificate(r.client, linseedCertLocation, common.OperatorNamespace())
+	if err != nil {
+		r.status.SetDegraded(operatorv1.ResourceReadError, fmt.Sprintf("Failed to retrieve / validate  %s", render.TigeraLinseedSecret), err, reqLogger)
+		return reconcile.Result{}, err
+	} else if linseedCertificate == nil {
+		log.Info("Linseed certificate is not available yet, waiting until they become available")
+		r.status.SetDegraded(operatorv1.ResourceNotReady, "Linseed certificate are not available yet, waiting until they become available", nil, reqLogger)
+		return reconcile.Result{}, nil
+	}
+
+	// Intrusion detection controller sometimes needs to make requests to outside sources. Therefore, we include
+	// the system root certificate bundle.
+	trustedBundle, err := certificateManager.CreateTrustedBundleWithSystemRootCertificates(managerInternalTLSSecret, esgwCertificate, linseedCertificate)
```

Comment:
Not sure we need `CreateTrustedBundleWithSystemRootCertificates`, as we never make requests externally. I believe `trustedBundle := certificateManager.CreateTrustedBundle(managerInternalTLSSecret, esgwCertificate, linseedCertificate)` should suffice.